### DISCUSSION TITLE
Welcome to Petari Discussions!

### DISCUSSION THREAD

**Discussion by Sahil1786** [Sahil1786/Petari#52] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kalyan9064** [comment]
hey please help me, i want to change the logo of this website, i didn't have that much experience. i can't access the root file, means i cant find indext.html file there. how can i access it please help me......

**Reply by Sahil1786** [reply]
You will find index.ejs inside the views folder.

**Comment by Saloni1707** [comment]
hey there i want to contribute to this project but m not able to pull request !!! 
or after assigning some task i would be able to do tht ???
